packages/web3-eth-accounts/src/index.js

```diff
@@ -228,9 +228,9 @@ Accounts.prototype.signTransaction = function signTransaction(tx, privateKey, ca
 
             var result = {
                 messageHash: '0x' + Buffer.from(signedTx.getMessageToSign(true)).toString('hex'),
-                v: '0x' + Buffer.from(signedTx.v).toString('hex'),
-                r: '0x' + Buffer.from(signedTx.r).toString('hex'),
-                s: '0x' + Buffer.from(signedTx.s).toString('hex'),
+                v: '0x' + signedTx.v.toString('hex'),
+                r: '0x' + signedTx.r.toString('hex'),
+                s: '0x' + signedTx.s.toString('hex'),
```

Comment:
We have a function `web3.utils.toHex` in the utilities and we should be using it consistently across the codebase instead of manually encoding. 

https://web3js.readthedocs.io/en/v1.5.0/web3-utils.html#tohex

Reply:
I think toHex is Megamorphic and its not usually recommended to go in that direction as its not aligned with inline cache.